src/CftfBundle/Resources/views/DocTree/export.json.twig

```diff
@@ -45,9 +45,12 @@
     ,"CFConcepts": [
     ]
     {%- endif %}
-    {% if licences is not empty -%}
-    {#- TODO: CFLicenses -#}
+    {% if licence is not empty -%}
     ,"CFLicenses": [
+        {
+            "title": {{ licence.title |json_encode |raw }}
+            , "licence_text": {{ licence.licenceText |json_encode |raw }}
```

Comment:
Should be `licenseText`, and are missing the other attributes required here: Ref: [IMS CASE Spec - CFLicense](https://www.imsglobal.org/sites/default/files/CASE/casev1p0/information_model/caseservicev1p0_infomodelv1p0.html#Interface_CFLicense)